### PR TITLE
Adding the ability to detect when undo_send is clicked + README updated

### PR DESCRIPTION
**Pull request by JerryBels**
Related to this : https://github.com/KartikTalwar/gmail.js/pull/305

### CODE REVIEW

**Comment by JerryBels** [comment]
Commited one more bug fix

**Comment by JerryBels** [comment]
Y U no merge

Am I doing things wrong ? Still a little unexperienced on github usage...

**Comment by JerryBels** [comment]
Am I doing things wrong @KartikTalwar ? Still a little inexperienced on github usage... Is this not able to be merged ?

File: src/gmail.js [hunk 1]
```diff
@@ -1655,7 +1655,7 @@ var Gmail_ = function(localJQuery) {
       page = hash;
     }
 
-    if(hash.indexOf('inbox/') !== -1) {
+    if(hash.indexOf('inbox/') !== -1 || hash.indexOf('sent/') !== -1 || hash.indexOf('all/') !== -1) {
       page = 'email';
```
Comment (josteink) [inline src/gmail.js]: Any particular reason this line is changed? Is it required to make undo_send work, or was it just meant as a general fix for another issue?

**Comment by josteink** [comment]
I think this looks like a good PR, except the concern I've outlined in my other comment.

Sometimes on Github, repo owners may be a bit busy/unresponsive. I usually start following up on my PRs if I haven't received any attention after around 1 week.

**Comment by josteink** [comment]
I think this looks like a good PR, except the concern I've outlined in my other comment.

Sometimes on Github, repo owners may be a bit busy/unresponsive. I usually start following up on my PRs if I haven't received any attention after around 1 week.

I'd also consider squashing the commits into one, so that it stick together and is easier to track/follow in the changelog if merged.

File: src/gmail.js (hunk 1 above)
Comment (JerryBels) [inline src/gmail.js]: It was a general fix yes... Should have commited separately ?

**Comment by JerryBels** [comment]
You're right, but part of it has already been merged, and after that @KartikTalwar asked me to fix the readme.

**Comment by josteink** [comment]
Fair enough. After this much waiting, I don't think we should be overly picky about minor details. 

I'll take a deeper, proper look at it later this evening and see if I can get it merged.

**Comment by JerryBels** [comment]
Thanks :)

**Pull request merged**